src/controllers/AbstractController.ts

```diff
@@ -0,0 +1,160 @@
+import { ApiPromise } from '@polkadot/api';
+import { BlockHash } from '@polkadot/types/interfaces';
+import { isHex } from '@polkadot/util';
+import { RequestHandler, Response, Router } from 'express';
+import * as express from 'express';
+import { BadRequest } from 'http-errors';
+import {
+	IAddressNumberParams,
+	IAddressParam,
+	INumberParam,
+} from 'src/types/request_types';
+
+import sanitizeNumbers from './utils/sanitizeNumbers';
+
+type SidecarRequestHandler =
+	| RequestHandler<IAddressParam>
```

Comment:
```suggestion
	RequestHandler<IAddressParam>
```

Reply:
Good eye!

Reply:
This is actually an eslint/prettier thing